app.js

```diff
@@ -0,0 +1,67 @@
+function animatedForm() {
+    const arrows = document.querySelectorAll(".fa-arrow-down");
+
+    arrows.forEach(arrow => {
+        arrow.addEventListener("click", () => {
+            const input = arrow.previousElementSibling;
+            const parent = arrow.parentElement;
+            const nextForm = parent.nextElementSibling;
+
+            //check for the validation
+            if(input.type === "text" && validateUser(input)){
+                nextSlide(parent,nextForm);
+            }else if(input.type == 'email' && validateEmail(input)){
+                nextSlide(parent,nextForm)
+            }else if(input.type == 'password' && validateUser(input)){
+                nextSlide(parent,nextForm)
+            }else{
+                parent.style.animation="shake 0.5s ease";
+            }
+            //get rid of animation
+            parent.addEventListener("animationend", () => {
+            parent.style.animation="";
+            });
+        });
+    });
+}
+
+

```

Comment:
@pravin1237 
Remove the extra lines. Make sure there is only a single line break after every function block.